app/jobs/state_file/import_from_direct_file_job.rb

```diff
@@ -35,6 +35,11 @@ def perform(authorization_code:, intake:)
         intake.synchronize_filers_to_database
 
         intake.update(df_data_import_succeeded_at: DateTime.now)
+
+        # removing duplicate associations here because sometimes we create duplicate records during data import
+        # future work will prevent this issue from happening and this can be removed
+        remove_duplicate_w2s(intake)
+        remove_duplicate_dependents(intake)
```

Comment:
I think i want to move these out of the `begin/rescue` block, like if the `destroy!` fails, idk if we want to make `df_data_import_errors` objects if we already set `df_data_import_succeeded_at` to a value.